modules/rules_xcodeproj/0.12.0/patches/module_version_bump.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
+new file mode 100644
+index 00000000..f14b8395
+--- /dev/null
++++ b/MODULE.bazel
+@@ -0,0 +1,22 @@
++module(
```

Comment:
nit: the name of this patch file is a bit confusing `module_version_bump.patch`, while the patch actually adds an entire MODULE.bazel file.

Reply:
Good point, updated.